### ISSUE TITLE
Mobile Meta Tag

### ISSUE THREAD

**Issue by Mikeysax** [MattW216/splurty#1]
On mobile things don't look as good as they could be.
One thing you could look into is to add the "mobile meta tag" to your app, so the responsive effect of Bootstrap actually shows up on Mobile.
Let me know in the comments once you figured that step out :)

**Comment by MattW216** [comment]
Hey Mike,
I just added the following meta tag in the head of my app:

<meta name="viewport" content="width=device-width, initial-scale=1">

I think viewing the quotes certainly looks much better now, whereas before, holding the phone in portrait mode would render the text too small, and landscape mode looked a little off as well.

However, it still doesn't look how I would like it too. I'm not sure how to prevent the index.html.erb content from bleeding into the footer. My idea for the design was to keep the header and footer backgrounds white, while having the quote background be the dark gray you see. But I can't seem to separate it. I tried adding a bit of padding-top to the .floater-blurb in master.scss but that didn't seem to do much.

I have my first mentor meeting tonight, and plan on asking about this. Please let me know what you think about the above edit so far though!

**Comment by MattW216** [comment]
Hmm, this is my first time commenting in GitHub, and it looks like it deleted the code I just added as part of my comment above. Adding it here, using the "Insert code" functionality:

`<meta name="viewport" content="width=device-width, initial-scale=1">`

**Comment by Mikeysax** [comment]
Have you tried this: https://v4-alpha.getbootstrap.com/getting-started/introduction/#responsive-meta-tag

As for things bleeding into the footer, you can try changing the position of the footer from absolute to relative. Otherwise, until you gain more skill with CSS, I recommend moving on and coming back later.

**Comment by MattW216** [comment]
Thanks for the link!

I took a look and bookmarked it for future reference. After reviewing, I added `shrink-to-fit=no` to my tag.

Also, just finished my mentor meeting and the way we tackled the bleeding issue was adding a background-color of white, to the footer - that way I could keep the footer absolute. I believe the app is now complete. Let me know what you think, and I'll close out this issue.
